webviz_subsurface/containers/_inplace_volumes.py

```diff
@@ -19,27 +19,33 @@ class InplaceVolumes(WebvizContainerABC):
     """### Volumetrics
 
 This container visualizes inplace volumetrics results from
-FMU ensembles. Input can be given either as aggregated csv files
-or as an ensemble name defined in 'container_settings' and csvfiles stored
-per realizations.
-In either case the csv files must follow FMU standards, that is it must have
-one or more of the following columns:
-'ZONE', 'REGION', 'FACIES', 'LICENSE' - these columns are used to filter the data.
+FMU ensembles.
+
+Input can be given either as aggregated csv files for volumes or or as an ensemble name
+defined in *container_settings* and volumetric csv files stored per realizations.
+
+####Volumetric input
+The volumetric csv files must follow FMU standards.
+[Example csv file](
+https://github.com/equinor/webviz-subsurface-testdata/blob/master/aggregated_data/volumes.csv)
+
+One of the columns: *ZONE*, *REGION*, *FACIES*, *LICENSE* must be present.
```

Comment:
One of these columns must be present: *ZONE*...

Is it at least one that is the requirement?